src/Braintree/Payment/OrderInformationService.php

```diff
@@ -81,19 +87,19 @@ public function extractLineItems(PaymentPayAction $payment): array
                 continue;
             }
 
-            // @TODO - We could potencially add more information here
             $lineItems[] = [
-                // 'commodityCode' => $this->substr($lineItem, 12),
-                // 'description' => $this->substr($lineItem->get, 127),
-                // 'discountAmount' => $lineItem,
+                'commodityCode' => $lineItem->getPayload()[self::LINE_ITEM_COMMODITY_CODE_CUSTOM_FIELD] ?? null,
```

Comment:
Shouldn't we ensure the length just in case someone exceeds the maximum with the custom value?

Reply:
I've removed the div inbetween